### PR TITLE
Migrate Search to Compose

### PR DESCRIPTION
**Pull request by simona-anomis**

### CODE REVIEW

File: Reply/app/src/main/java/com/materialstudies/reply/ui/search/SearchBar.kt [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import com.materialstudies.reply.util.DevicePreviews
+import com.materialstudies.reply.util.ThemePreviews
+
+@Composable
+fun SearchBar(
+    modifier: Modifier = Modifier,
+    onBackClicked: () -> Unit = {}
+) {
+    var inputText by remember { mutableStateOf("") }
+    val borderColor = MaterialTheme.colors.onSurface
```
Comment (arriolac) [inline Reply/app/src/main/java/com/materialstudies/reply/ui/search/SearchBar.kt:54]: Any reason to now use this inline instead?

File: Reply/app/src/main/java/com/materialstudies/reply/ui/search/SearchBar.kt [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        .drawBehind {
+            drawLine(
+                color = borderColor,
+                start = Offset(x = 0F, y = size.height),
+                end = Offset(x = size.width, y = size.height),
+                strokeWidth = 0.5F
+            )
+        }
+        .padding(horizontal = dimensionResource(id = R.dimen.grid_2)),
+        verticalAlignment = Alignment.CenterVertically) {
```
Comment (arriolac) [inline Reply/app/src/main/java/com/materialstudies/reply/ui/search/SearchBar.kt:66]: Nit formatting:
.padding(
    horizontal = dimensionResource(id = R.dimen.grid_2),
    verticalAlignment = Alignment.CenterVertically
) { ... }

**Review by arriolac — APPROVED** [review]
LGTM just minor nits

File: Reply/app/src/main/java/com/materialstudies/reply/ui/search/SearchBar.kt (hunk 2 above)
Comment (simona-anomis) [reply Reply/app/src/main/java/com/materialstudies/reply/ui/search/SearchBar.kt:66]: verticalAlignment isn't a param of padding, it's a param of the outer Row

File: Reply/app/src/main/java/com/materialstudies/reply/ui/search/SearchBar.kt (hunk 1 above)
Comment (simona-anomis) [reply Reply/app/src/main/java/com/materialstudies/reply/ui/search/SearchBar.kt:54]: MaterialTheme.colors.onSurface is a @Composable invocation, so it needs a  @Composable context, which drawLine() isnt

**Pull request merged**